src/ixdat/techniques/reactor.py

```diff
@@ -0,0 +1,289 @@
+
+            elif original_unit == "mbar":
+                unit_factor = {
+                    "mbar": 1,
+                    "bar": 1000,
+                    "hPa": 1,
+                    "kPa": 0.1,
+                }[new_unit]
+
+            elif original_unit == "bar":
+                unit_factor = {
+                    "mbar": 1e-3,
+                    "bar": 1,
+                    "hPa": 1e-3,
+                    "kPa": 0.1e-3,
+                }[new_unit]
+
+            else:
+                unit_factor = {
+                    # Time conversion
+                    "s": 1,
+                    "min": 1 / 60,
+                    "minutes": 1 / 60,
+                    "h": 1 / 3600,
+                    "hr": 1 / 3600,
+                    "hour": 1 / 3600,
+                    "hours": 1 / 3600,
+                    "d": 1 / (3600 * 24),
+                    "days": 1 / (3600 * 24),
+                    # Pressure conversion
+                    "mbar": 1,
+                    "bar": 1000,
+                    "hPa": 1,
+                    "kPa": 0.1,
+                    # Temperature conversion
+                    "K": 273.15,
+                    "kelvin": 273.15,
+                }[new_unit]
+        except KeyError:
+            warnings.warn(
+                f"Can't convert original unit '{original_unit}' to new unit"
+                f"'{new_unit}'. You can try to clear cache with "
+                "measurement.clear_cache(). Using original unit {original_unit} !",
+                stacklevel=2,
+            )
+            unit_factor = 1
+            new_unit = original_unit
+        return unit_factor, new_unit
+
+
+class SpectroReactorMeasurement(ReactorMeasurement, SpectroMSMeasurement):
+    default_plotter = SpectroTPMSPlotter
+
+
+class ReactorUnitConverterResult(Saveable):
+    """A class for a reactor reactor_calibration result.
+
+    FIXME: I think that something inheriting directly from Saveable does not belong in
+        a technique module.
+    """
+
+    table_name = "reactor_cal_results"
+    column_attrs = {"name", "mass", "new_unit"}
+
+    def __init__(
+        self,
+        name=None,
+        mass=None,
+        new_unit=None,
+    ):
+        super().__init__()
+        self.name = name or f"{mass}@{new_unit}"
+        self.mass = mass
+        self.new_unit = new_unit
+
+    def __repr__(self):
+        return (
+            f"{self.__class__.__name__}(name={self.name}, mol={self.mass}, "
+            f"calibration type={self.cal_type})"
+        )
+
+    # @property
+    # def color(self):
+    #    return STANDARD_COLORS[self.mass]
+
+
+class ReactorCalibration(Calibration):
+    """A reactor calibration to calibrate inverse of meta_series"""
+
+    extra_column_attrs = {
+        "reactor_calibration_results": ("reactor_cal_results", "reactor_cal_result_id")
+    }
+    # TODO: https://github.com/ixdat/ixdat/pull/11#discussion_r677552828
+
+    def __init__(
```

Comment:
Is this `__init__` needed? It seems not to add anything to `Calibration.__init__` except for `new_unit` which I don't see used in `calibrate_series`.
`calibrate_series` is the only method of a `Calibration` which is actually needed.

Reply:
I have removed the __init__(). it came from earlier thoughts on how to calibrate copied from ms calibration.